### ISSUE TITLE
Comment out the project URLs configured in `setup.cfg`

### ISSUE THREAD

**Issue by DragaDoncila** [napari/cookiecutter-napari-plugin#127]
In #32 we added defaults for project URLs in `setup.cfg`. We've since had some feedback that developers don't really know where that information comes from on the napari hub, and how to change it. At the very least, we should comment out these URLs so that plugins don't show information that hasn't been explicitly exposed by the plugin developer.

**Comment by Czaki** [comment]
I rather prefer to add some detailed documentation than remove this. This is one of the core python packaging features.

**Comment by DragaDoncila** [comment]
@Czaki I agree that it's a core feature and we should encourage people to provide these URLs - I added them in the first place! However, what we're finding is that people are not aware of these URLs, and they're not editing them. This is despite clear instructions after creation that you should go and edit these. So I don't think it's a matter of more detailed documentation.

My hope is that we can move more towards **conscious and explicit** actions from developers to provide appropriate metadata and documentation for their plugins, rather than us providing them with default values. By commenting these values out, developers will be alerted to this lack of metadata on the PyPI project page, their napari hub plugin page and the napari hub preview page. This will allow them to go and make the conscious choice of adding these URLs back in, and considering where they point to, rather than having "complete" metadata they are unaware of.

**Comment by brisvag** [comment]
Good point @DragaDoncila, if this comes with a "plugin completeness" warning of some sort from the hub, I agree with commenting them out.

**Comment by Czaki** [comment]
> developers will be alerted to this lack of metadata on the PyPI project page

I worry that they will not notice this. But in general I agree with @brisvag that with Warning on napari hub it may be enough.
